lib/mix/tasks/nerves_hub.product.ex

```diff
@@ -41,6 +41,49 @@ defmodule Mix.Tasks.NervesHub.Product do
   Change product name
 
       mix nerves_hub.product update example name example_new
+
+  # Managing user roles
+
+  The following functions allow the management of user roles within your product.
+  Roles are a way of granting users a permission level so they may perform
+  actions for your product. The following is a list of valid roles in order of
+  highest role to lowest role:
+
+    :admin
```

Comment:
I think these should be listed here in string form since if you actually pass the colon in on the commandline it won't work.... or will it?